contrib/cirrus/runner.sh

```diff
@@ -52,19 +58,37 @@ _run_verify_vendor() {
     fi
 }
 
+_run_verify_vendor_aarch64() {
+    _run_verify_vendor
+}
+
 _run_unit() {
     make unit
 }
 
+_run_unit_aarch64() {
+    _run_unit
+}
+
 _run_integration() {
     make integration
 }
 
+_run_integration_aarch64() {
+    _run_integration
+}
+
 show_env_vars
 
 msg "************************************************************"
 msg "Toolchain details"
 msg "************************************************************"
+
+# FIXME (@lsm5): rustup commands should be moved to c/automation_images repo
+# https://github.com/containers/automation_images/issues/150
+rustup install stable
+rustup default stable
```

Comment:
I noticed in the Makefile for the old cross target, we were doing a:

`rustup target add aarch64-unknown-linux-gnu`, maybe that's what we're missing for arm?